sdk/eventhub/event-hubs/karma.conf.js

```diff
@@ -105,7 +105,7 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: false,
+    singleRun: true,
```

Comment:
What is the story behind this? Or something we just missed?

Reply:
I believe this is just something we missed. You would want this set to `false` if you were debugging the tests in the browsers, but for CI/normal case we'd want this set to `true`. Otherwise it continues running in the background even after the process appears to exit. (This is fine for CI since it moves onto the next job, but locally you see the tests just trying to run again periodically)

Reply:
Can we make sure that the karma config in the template project reflects the right value?